src/lib/viewers/controls/rotate/RotateControl.scss

```diff
@@ -0,0 +1,19 @@
+@import '../styles';
+
+.bp-RotateControl {
+    @include bp-ControlGroup;
+}
+
+.bp-RotateControl-button {
+    @include bp-ControlButton($width: 32px);
```

Comment:
Is this the width of the button in the current experience?

Reply:
I'm going to check it visually and see the difference.